spec/features/user_can_like_a_post_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature "Like a post", type: :feature do
+
+  before(:each) do
+    successful_sign_up
+    first_user_post
+  end
+
+  scenario "A user can like a post" do
+    click_button "Like"
+    expect(page).to have_content('test_first_name My first post less than a minute')
+  end
```

Comment:
is this test testing the like feature?

Reply:
It will change when we do the count of likes